xrpl/model/transactions/payment.go

```diff
@@ -93,7 +93,7 @@ func (p *Payment) Flatten() FlatTransaction {
 	}
 
 	if p.DestinationTag != 0 {
-		flattened["DestinationTag"] = p.DestinationTag
+		flattened["DestinationTag"] = int(p.DestinationTag)
```

Comment:
Would it make sense to type it as Uint32 as it's what's written in the docs ?
https://xrpl.org/docs/references/protocol/transactions/types/payment#payment-fields

It's also typed as uint32 in the Payment model in the code above.